dae/dae/backends/cnv/tests/test_flexible_cnv_loader.py

```diff
@@ -0,0 +1,247 @@
+import textwrap
+import pytest
+import io
+
+from dae.utils.variant_utils import mat2str
+from dae.genomic_resources.test_tools import convert_to_tab_separated
+from dae.pedigrees.loader import FamiliesLoader
+
+from dae.variants.core import Allele
+from dae.backends.cnv.flexible_cnv_loader import flexible_cnv_loader
+
+
+@pytest.fixture
+def families():
+    ped_content = io.StringIO(convert_to_tab_separated(textwrap.dedent(
+        """
+            familyId personId dadId	 momId	sex status role
+            f1       f1.m     0      0      2   1      mom
+            f1       f1.d     0      0      1   1      dad
+            f1       f1.p1    f1.d   f1.m   1   2      prb
+            f1       f1.s2    f1.d   f1.m   2   1      sib
+            f2       f2.m     0      0      2   1      mom
+            f2       f2.d     0      0      1   1      dad
+            f2       f2.p1    f2.d   f2.m   2   2      prb
+        """)))
+    families = FamiliesLoader(ped_content).load()
+    assert families is not None
+    return families
+
+
+@pytest.fixture
+def genome(gpf_instance_2013):
+    return gpf_instance_2013.reference_genome
+
+
+@pytest.mark.parametrize(
+    # params = (cnv_family_id, cnv_location, cnv_variant_type, cnv_best_state)
```

Comment:
commented-out code

Reply:
this is not a code, but a comment for the structure of the params tuple;